src/services/item/plugins/published/service.ts

```diff
@@ -114,6 +115,36 @@ export class ItemPublishedService {
     };
   }
 
+  private async getPublishedIfExist(item: Item, itemPublishedRepository: ItemPublishedRepository) {
+    try {
+      return await itemPublishedRepository.getForItem(item);
```

Comment:
```suggestion
      return itemPublishedRepository.getForItem(item);
```

Reply:
good catch ! thanks

Reply:
I'm not sure to understand why I have to remove the `await` here. Especially in this case, if I don't wait on `getForItem`, I'll not catch the exception, am I wrong ?